README.md

```diff
@@ -6,8 +6,59 @@ If you have any feedback or are interested in what we are doing please [contact
 
 API for managing and accessing Ably Spaces.
 
+**Class definitions are available [here](/docs/class-definitions.md).**
+
 ## Spaces
 
 A Space is an Ably concept that allows for shared realtime digital experiences such as chat, reactions and collaborative editing.
 
 A Space consists of users and storage with persistent presence.
+
+### Usage
+
+To use the Space API, you'll need to have an Ably client already set up.
+Here's an example of how you can create a new instance of the client and pass it to the Spaces constructor:
+
+```ts
+import { Realtime } from 'ably/promise';
+import Spaces from '@ably-labs/spaces';
+
+const client = new Realtime(process.env.ABLY_KEY);
```

Comment:
I think it might be a bit confusing to use `process.env.ABLY_KEY` here as this is browser library? Perhaps simply a string "ABLY_API_KEY".